uint32 counter
float32 current
float32 shaft_angle
float32 shaft_velocity
